src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+import Duke.Action;
+import Duke.InputNotValidError;
+import Duke.Storage;
+import Duke.TaskList;
+import Duke.Ui;
+import Duke.Parser;
+
+ /**
+  * <pre>
+  * Duke class, which is the main class
+  * to run this duke app.
+  * </pre>
+  * @param <Storage> storage object stoating history
+  * @param <TaskList> tasklist object to sort tasks
+  * @author Yuichiro Fukushima
+  */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private Storage storage;
+    private TaskList tasklist;
```

Comment:
Perhaps you should change the name to taskList to comply with the coding standards!